tracing-futures/src/lib.rs

```diff
@@ -64,6 +64,9 @@ pub trait Instrument: Sized {
     fn instrument(self, span: Span) -> Instrumented<Self> {
         Instrumented { inner: self, span }
     }
+    fn current_instrument(self) -> Instrumented<Self> {
```

Comment:
What about 
```suggestion
    fn in_current(self) -> Instrumented<Self> {
```
or
```suggestion
    fn in_current_span(self) -> Instrumented<Self> {
```

The second might communicate intent more clearly, but is also a bit longer...